modules/machineset-modifying.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// 
+// * machine_management/modifying-machineset.adoc
+
+[id="machineset-modifying_{context}"]
+= Modifying a MachineSet
+
+When making changes to a MachineSet, you must remove all machines associated with the MachineSet 
+by deleting or scaling down the machines to `0`. Then scale the replicas back to the desired number. 
+Changes you make to a MachineSet do not affect existing machines. 
+
+If you need to scale a MachineSet without making other changes, you do not need
+to delete the machines.
+
+[NOTE]
+====
+By default, the {product-title} router pods are deployed on workers. Because the router 
+is required to access some cluster resources, including the web console, 
+do not scale the worker MachineSet to `0` unless you first relocate the router pods.
+====
+
+.Prerequisites
+
+* Install an {product-title} cluster and the oc command line.
+* Log into `oc` as a user with `cluster-admin` permission.
+
+.Procedure
+
+. Edit the MachineSet:
++
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
+
+. Scale down the MachineSet to `0`:
++
+----
+$ oc scale --replicas=0 machineset <machineset> -n openshift-machine-api
+----
+Or:
++
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
++
+Wait for the machines to be removed.
++
+. Scale up the MachineSet as needed:
++
+----
+$ oc scale --replicas=2 machineset <machineset> -n openshift-machine-api
+----
+Or:
++
+----
+$ oc edit machineset <machineset> -n openshift-machine-api
+----
++
+Wait for the machines to start. The new machines use the updated MachineSet. 
```

Comment:
- `s/machines/Machines`
- The new Machines contain changes made to the Machineset's Machine template.

Reply:
looks good